crates/assembly/src/assembler.rs

```diff
@@ -206,16 +206,16 @@ impl Assembler {
     /// use miden_assembly::{Assembler, Path};
     ///
     /// let mut assembler = Assembler::default();
-    /// assembler.compile_and_statically_link_from_dir("~/masm/std", "std::foo");
+    /// assembler.compile_and_statically_link_from_dir("~/miden/core/std", "miden::core::foo");
```

Comment:
The example of `miden/core/std` is somewhat confusing here (and in a few other places in this file). Should it have been just `masm/core`?